package.json

```diff
@@ -78,10 +78,7 @@
     "fix": "npm run lint:style -- -w && npm run lint:code -- --fix",
     "prepare": "npm run build",
     "prepublishOnly": "npm run lint",
-    "test": "npm run build && npm run test:14 && npm run test:12 && npm run test:10 && npm run test:host",
-    "test:10": "node test/test.js node10 no-npm",
-    "test:12": "node test/test.js node12 no-npm",
-    "test:14": "node test/test.js node14 no-npm",
+    "test": "npm run build && npm run test:host",
```

Comment:
should this be updated instead to run tests against 16, 18 and 19? I'm not exactly sure what is the goal of this existing infrastructure to test on a matrix of node versions in the runner, and run test scripts targeting specific node versions...cross-target compatibility maybe?

Reply:
ah probably, i will limit it to the latest nodejs to test older versions
it will speed up things i guess